config/applitools.config.js

```diff
@@ -0,0 +1,44 @@
+const { execSync } = require('child_process');
+const merge = require('lodash/merge');
```

Comment:
You need to move `lodash` to your `dependencies`, it's currently in `devDependencies`